src/lib/daemon_rpcs/daemon_rpcs.ml

```diff
@@ -201,28 +203,37 @@ module Types = struct
       let sync_status = map_entry "Sync Status" ~f:Sync_status.to_string
 
       let propose_pubkeys =
-        map_entry "Block Producers Running" ~f:(fun keys ->
+        map_entry "Block producers running" ~f:(fun keys ->
```

Comment:
don't need to fix now, but can probably clean this up later to not have `Total: ` and omit the () if there are no block producers. Same with `peers`.